Podcast/PrivacyPolicyViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  PrivacyPolicyViewController.swift
+//  Podcast
+//
+//  Created by Jack Thompson on 3/5/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class PrivacyPolicyViewController: UIViewController, UIWebViewDelegate {
```

Comment:
make this a subclass of `ViewController` instead of `UIViewController` and make the `mainScrollView = webView`